test/presenters/return-requirements/frequency-collected.presenter.test.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const FrequencyCollectedPresenter = require('../../../app/presenters/return-requirements/frequency-collected.presenter.js')
+
+describe('Frequency Collected presenter', () => {
+  let session
```

Comment:
Same comment as other PRs. The session can be declared a describe block down where it is within the context of that describe block.